config/locales/en.yml

```diff
@@ -53,8 +53,22 @@ en:
               greater_than: Other income must be greater than £0
         outgoings_form:
           attributes:
-            outgoings:
-              at_least_one_item:  Please select at least one option
+            housing_payments_value:
+              blank: Please enter a 0 if your client makes no housing payments
```

Comment:
Is this copy from anywhere? I note that we use "enter 0" in the screengrab and "enter zero" in the additional acceptance criteria for the top of the page guidance. I can't see these error strings defined elsewhere but I just note that "a 0" struck me as odd in that it was different. It's entirely possible I'm just missing where this was designed